fastlane/Fastfile

```
@@ -0,0 +1,84 @@
+default_platform(:android)
+
+platform :android do
+  desc "Build Android Debug APK"
+  lane :build_debug do
+    gradle(
+      project_dir: ".",
+      task: "assembleDebug",
+      properties: { "katana.flavor" => "dev" },
```

RuboCop (1.85.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)